view/frontend/templates/oneclick/magiccart.phtml

```diff
@@ -46,5 +46,6 @@ $cartDataCount = count($cartData);
     </div>
     <link rel="stylesheet" type="text/css"
           href="<?php echo $block->getViewFileUrl('Razorpay_Magento::css/magic.css'); ?>">
-
+    <script type="text/javascript" src="<?php echo $block->getViewFileUrl('Razorpay_Magento::js/magic-cart-vanilla.js'); ?>">
```

Comment:
1. `type="text/javascript"` is redundant,
2. Use quick echo
3. Add missing escaping.

```suggestion
    <script src="<?= $escaper->escapeUrl($block->getViewFileUrl('Razorpay_Magento::js/magic-cart-vanilla.js')) ?>">
```